entity-service/src/test/kotlin/com/egm/stellio/entity/web/EntityHandlerTests.kt

```diff
@@ -183,7 +183,7 @@ class EntityHandlerTests {
 
         webClient.post()
             .uri("/ngsi-ld/v1/entities")
-            .header("Link", aquacHeaderLink)
+            .header(HttpHeaders.LINK, aquacHeaderLink)
```

Comment:
Maybe we can add new tests for handling the Accept header

Reply:
Yes, we should add a few